src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/Management/StudioServiceClient.cs

```diff
@@ -0,0 +1,243 @@
+                if (_httpResponse.Headers.Contains("x-ms-request-id"))
+                {
+                    ex.RequestId = _httpResponse.Headers.GetValues("x-ms-request-id").FirstOrDefault();
+                }
+                if (_shouldTrace)
+                {
+                    ServiceClientTracing.Error(_invocationId, ex);
+                }
+                _httpRequest.Dispose();
+                if (_httpResponse != null)
+                {
+                    _httpResponse.Dispose();
+                }
+                throw ex;
+            }
+            // Create Result
+            var _result = new AzureOperationResponse<Azure.Management.MachineLearning.WebServices.Models.WebService>();
+            _result.Request = _httpRequest;
+            _result.Response = _httpResponse;
+            if (_httpResponse.Headers.Contains("x-ms-request-id"))
+            {
+                _result.RequestId = _httpResponse.Headers.GetValues("x-ms-request-id").FirstOrDefault();
+            }
+            // Deserialize Response
+            if ((int)_statusCode == 200)
+            {
+                _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
+                try
+                {
+                    _result.Body = SafeJsonConvert.DeserializeObject<Azure.Management.MachineLearning.WebServices.Models.WebService>(_responseContent, this._client.DeserializationSettings);
+                }
+                catch (JsonException ex)
+                {
+                    _httpRequest.Dispose();
+                    if (_httpResponse != null)
+                    {
+                        _httpResponse.Dispose();
+                    }
+                    throw new SerializationException("Unable to deserialize the response.", _responseContent, ex);
+                }
+            }
+            if (_shouldTrace)
+            {
+                ServiceClientTracing.Exit(_invocationId, _result);
+            }
+            return _result;
+        } 
+
+        /// <summary>
+        /// Gets a web service definition
+        /// </summary>
+        /// Retrive the web service definition from an experiement.
+        /// <param name='operations'>
+        /// The operations group for this extension method.
+        /// </param>
+        /// <param name='workspaceId'>
+        /// The id of the workspace where the experiment resides.
+        /// </param>
+        /// <param name='experimentId'>
+        /// The id of the experiment within the workspace referred to by workspaceId.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        public async Task<Microsoft.Azure.Management.MachineLearning.WebServices.Models.WebService> GetAsync(string workspaceId, string experimentId, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
The method name seems too generic to me. Please update it with a specific one, e.g. GetWebServiceDefinitionAsync()